[INFO] Starting dashboard generation
[INFO] Loading Excel data for Shawn...
[INFO] Loaded Shawn rows: 4054
[INFO] Loading Excel data for Barry...
[INFO] Loaded Barry rows: 4054
[INFO] Preprocessing Shawn...
c:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\Visualizor.py:1203: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df["Completed Date"] = pd.to_datetime(df["Completed Date"], errors="coerce")
[INFO] Finished preprocessing Shawn. Remaining rows: 3993
[INFO] Preprocessing Barry...
c:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\Visualizor.py:1203: SettingWithCopyWarning:
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  df["Completed Date"] = pd.to_datetime(df["Completed Date"], errors="coerce")
[INFO] Finished preprocessing Barry. Remaining rows: 3993
[INFO] Assembling full HTML output...
[INFO] Generating HTML for customer: shawn
C:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\.venv\Lib\site-packages\openpyxl\worksheet\_reader.py:223: UserWarning:

Cell AQ2036 is marked as a date but the serial value 553420928 is outside the limits for dates. The cell will be treated as an error.

C:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\.venv\Lib\site-packages\openpyxl\worksheet\_reader.py:329: UserWarning:

Data Validation extension is not supported and will be removed

[DEBUG] GlobalTab ID exists: True
[INFO] Generating HTML for customer: barry
C:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\.venv\Lib\site-packages\openpyxl\worksheet\_reader.py:223: UserWarning:

Cell AQ2036 is marked as a date but the serial value 553420928 is outside the limits for dates. The cell will be treated as an error.

C:\Users\AD39644\Desktop\EIS\OneDrive_2024-07-25\Auditing Script\.venv\Lib\site-packages\openpyxl\worksheet\_reader.py:329: UserWarning:

Data Validation extension is not supported and will be removed

[DEBUG] GlobalTab ID exists: True
